### DISCUSSION TITLE
Jedis ping method deprecated from JedisCluster

### DISCUSSION THREAD

**Discussion by ashishkeshu** [xetorthio/jedis#2629] [Q&A]
Dear Team,

I am using below constructor to create jedis cluster.
new JedisCluster(jedisClusterNode, connTimeout, soTimeout, maxAttempts, username, password, "", jedisPoolConfig, false);

there is no exception thrown when server nodes are not reachable. how to identify if connection with redis server was successful or not.
Earlier ping command could be used to check the connection status but that too is removed.

Can you please tell what is the appropriate mechanism to check if the redis cluster is successfully connected to redis server.
it will be really helpful.

Thanks,
Ashish

**Comment by sazzad16** [comment]
Resolved by #2721 
Released in 4.0.0
